src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md

```diff
@@ -84,12 +84,22 @@ system environment variables with the same name.
 
 ## Environment variables k6 will read automatically
 
-k6 will also try to read some specific environment variables that the user can set to change the
-behavior of k6:
+k6 will also try to read some specific environment variables that the user can
+set to change the behavior of k6. These must be supplied to k6 at the system level
+(so not using `-e` or `--env`).
 
 | Name                 | Description                                                                                                            |
 | -------------------- | ---------------------------------------------------------------------------------------------------------------------- |
 | `K6_CLOUD_HOST`      | A URL to connect to, when the --out=cloud results output option is specified.                                          |
 | `K6_CLOUD_TOKEN`     | An authentication token to use in API calls to the cloud service, when the --cloud results output option is specified. |
 | `K6_NO_USAGE_REPORT` | Define this to disable [usage reports](/misc/usage-collection).                                                        |
 | `K6_OUT`             | Specify results output, same as --out command-line option.                                                             |
+
+k6 also allows proxying via its use of [Golang's standard http library](https://pkg.go.dev/net/http#ProxyFromEnvironment).
+This is useful e.g. when wanting to send data to [k6 cloud](/cloud) from a k6 running inside a restricted network.
+
+| Name                 | Description                                                          |
+| -------------------- | ---------------------------------------------------------------------|
+| `HTTP_PROXY`         | Given either as complete URL or as `host:port` for HTTP requests.    |
+| `HTTPS_PROXY`        | Given either as complete URL or as `host:port` for HTTPS requests.   |
+| `NO_PROXY`           | Don't use any proxy for requests.                                    |
```

Comment:
Maybe we should expand on the values for NO_PROXY based on https://pkg.go.dev/golang.org/x/net/http/httpproxy#Config

Reply:
Yes that makes sense.